src/components/DataCard/PieGraph.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { PieChart, Pie, Cell, Legend, ResponsiveContainer } from "recharts";
+import { pieColors } from "../../Utils/Constances";
+import PieItem from "./PieItem";
+import Articles from "../../mockData/mockData.json"
+
+const PieGraph = () => {
+  // local data - need to change after connect to real data
+  const data = [
+    { name: "NBC", value: 400 },
+    { name: "Vulture", value: 300 },
+    { name: "CNN", value: 300 },
+    { name: "ESPN", value: 200 },
+    { name: "BBC", value: 200 },
+  ];
+  const COLORS = [
+    pieColors.black,
+    pieColors.orange,
+    pieColors.lightAzure,
+    pieColors.darkBlue,
+    pieColors.gray,
+  ];
+
+  const renderGraphColors = () => {
+    return data.map((entry, index) => (
+      <Cell key={`cell-${index}`} fill={COLORS[index % COLORS.length]} />
+    ));
+  };
+
+  const renderPieItem = () => {
+    const fourElement = data.slice(0,4); 
```

Comment:
why do you limit the array?

Reply:
to show only the top four sources in the data card (like in figma)